src/core/mac/mac.hpp

```diff
@@ -304,6 +304,23 @@ class Mac: public ThreadNetifLocator
     void SetBeaconEnabled(bool aEnabled) { mBeaconsEnabled = aEnabled; }
 
     /**
+     * This method gets the Joining Permitted flag used in any subsequently transmitted IEEE 802.15.4 Beacon frames.
+     *
+     * @retval TRUE   if the Joining Permitted flag will be set in Beacon frames.
+     * @retval FALSE  if the Joining Permitted flag will not set in Beacon frames.
+     *
+     */
+    bool GetBeaconJoinableFlag(void)  { return mBeaconJoinableFlag; }
```

Comment:
nits:
- change `Joinable` to `JoiningPermitted`?  Thread Spec calls this the "Joining Permitted" bit.
- declare as `const`